CHANGELOG.md

```diff
@@ -72,6 +72,38 @@ Visit our [1.0.0 migration guide](https://github.com/mapbox/mapbox-navigation-an
 * Fix missing destination marker regression when using the Drop-in UI [#3462](https://github.com/mapbox/mapbox-navigation-android/pull/3462)
 * Updated Nav UI day and night styles to new stable production v1 versions [#3520](https://github.com/mapbox/mapbox-navigation-android/pull/3520)
 
+## Mapbox Navigation SDK 1.0.0 - October 2, 2020
+Today, we are releasing Navigation SDK (Core & UI components) v.1.0 for Android 🎉 🚀 
+
+### Core Components
+This upgrade features 
+- a higher accuracy location engine which functions even in low GPS quality scenarios such as tunnels or overpasses, 
+- free-drive mode which enables accurate location positioning even without an active route for daily commuting, 
+- a new modular architecture that allows developers to customize the navigation experience for their use case, 
+- and MAUs-based billing that provides predictability in costs to developers.
+
+### UI Components
+This release as compared to legacy offers all the features but with much finer control and granularity. It mainly serves as a port of the legacy UI SDK implementation to use the `1.0` version of the Navigation SDK (Core components) and its features. The `1.0` UI Components also removes redundant methods & APIs while exposing new ones instead. It brings in new features, including:
+
+- Two different ways of providing feedback during a trip session, thereby helping Mapbox provide better route quality, turn-by-turn experiences, traffic congestion, etc.
+- Allowing developers to deemphasize portions of the route line behind the puck, thereby reflecting route progress state.
+- Providing UI components that visualize a single building footprint or extrusion. Great to use for marking the final destination in an improved arrival experience.
+
+### Pricing
+When routes are requested with `MapboxNavigation#requestRoutes`, usage of Mapbox APIs is now [billed](https://www.mapbox.com/pricing/#navmaus) together based on [monthly active users](https://docs.mapbox.com/help/glossary/monthly-active-users/) rather than individually by HTTP request. Learn more in the [pricing by product documentation](https://docs.mapbox.com/accounts/overview/pricing/#navigation-sdk-for-ios-and-android). If you prefer to still use request-based billing, inject the routes with `MapboxNavigation#setRoutes` or `NavigationViewOptions.Builder#directionsRoute` instead.
```

Comment:
Please remove the section : If you prefer to still use request-based billing, inject the routes with `MapboxNavigation#setRoutes` or `NavigationViewOptions.Builder#directionsRoute` instead.

Also add the following text (instead of linking to the page)

Applications built with v1.0.0+ are billed based only on monthly active users, specifically "Navigation SDKs" MAUs. Navigation SDK MAUs include Directions API, Vector Tiles API, and Raster Tiles API requests with no upfront commitments or annual contracts. A Navigation SDK MAU becomes a billable event only once a user utilizes the Navigation SDK for turn-by-turn directions or for free-drive.

A single user is billed as one MAU across app upgrades as long as the app is not deleted. Deleting and re-installing an app that uses the Navigation SDK would result in an additional MAU. This happens because the SDK does not store personally identifying information.

To see the number of Navigation SDKs MAUs included in the free tier and the cost per Navigation SDKs MAU beyond the free tier, see the Navigation SDKs section of our pricing page.